cli/cmd/root.go

```diff
@@ -118,35 +85,11 @@ func NewCmdRoot() *cobra.Command {
 	cmd.PersistentFlags().StringVar(&logFileName, "log-file-name", "cloudquery.log", "Log filename")
 
 	// Telemtry (analytics) flags
-	cmd.PersistentFlags().Bool("no-telemetry", false, "disable telemetry collection")
-	// we dont need viper support for most flags as all can be used via command line for now (we can add in the future if really necessary)
-	// the only exception is the telemetry as people might want to put in a bash starter script
-	if err := viper.BindPFlag("no-telemetry", cmd.PersistentFlags().Lookup("no-telemetry")); err != nil {
-		panic(err)
-	}
-	cmd.PersistentFlags().Bool("telemetry-inspect", false, "enable telemetry inspection")
-	cmd.PersistentFlags().Bool("telemetry-debug", false, "enable telemetry debug logging")
-
-	// Sentry (error reporting) flags
-	cmd.PersistentFlags().StringVar(&sentryDsn, "sentry-dsn", sentryDsnDefault, "sentry DSN")
-
-	hiddenFlags := []string{"telemetry-inspect", "telemetry-debug", "sentry-dsn"}
-	for _, f := range hiddenFlags {
-		err := cmd.PersistentFlags().MarkHidden(f)
-		if err != nil {
-			panic(err)
-		}
-	}
-	initViper()
+	cmd.PersistentFlags().Var(telemetryLevel, "telemetry-level", "Telemetry level (none, errors, all)")
```

Comment:
There is some inconsistency here between `none, errors, all` and the enum on line 29 https://github.com/cloudquery/cloudquery/pull/2619/files#diff-eb154bbd601602eeae895e7af6a12b88cbdb7d585a2d32216c398f68cabcdff5R29 (`"trace", "debug", "info", "warn", "error"`). I assume we want the enum to be `none, errors, all`? I will push a fix for this